storage/src/testFixtures/java/tech/pegasys/teku/storage/storageSystem/FileBackedStorageSystemBuilder.java

```diff
@@ -49,13 +49,28 @@ public class FileBackedStorageSystemBuilder {
   private boolean storeVotesEquivocation = false;
   private boolean storeBlockExecutionPayloadSeparately = false;
 
+  private int blockMigrationBatchSize = 25;
+
   private FileBackedStorageSystemBuilder() {}
 
   public static FileBackedStorageSystemBuilder create() {
     return new FileBackedStorageSystemBuilder();
   }
 
   public StorageSystem build() {
+    final Database database = buildDatabaseOnly();
+
+    validate();
+    return StorageSystem.create(
+        database,
+        createRestartSupplier(),
+        storageMode,
+        storeConfig,
+        spec,
+        ChainBuilder.create(spec));
+  }
+
+  public Database buildDatabaseOnly() {
```

Comment:
this is only helpful in a pretty specific case, because the database won't be in a state to standup the storage system until hot blocks are migrated... so for the database migration test its important to be able to get access to the database directly, so that migration can be run.